lib/tasks/complete_technical_failure_emails.rake

```diff
@@ -0,0 +1,9 @@
+namespace :complete_technical_failure_emails do
+  desc "Update emails marked as technical failure to have a completed_at time"
+  task fill_completed_at_time: :environment do
+    incomplete_technical_failure_emails = Email.where(failure_reason: "technical_failure").where(finished_sending_at: nil)
```

Comment:
```suggestion
    incomplete_technical_failure_emails = Email.where(failure_reason: :technical_failure, finished_sending_at: nil)
```

This might be slightly tidier.